src/_alfasim_sdk/alfacase/generate_case_description_docstring.py

```diff
@@ -33,6 +33,56 @@
 """ Space needed to reach the root of the literal block. """
 BASE_INDENT = " " * 12
 
+CATEGORIES_USED_ON_DESCRIPTION = [
+    "time",
+    "length",
+    "temperature",
+    "volume flow rate",
+    "volume fraction",
+    "mass flow rate",
+    "standard volume per standard volume",
+    "pressure",
+    "mass fraction",
+    "mole per mole",
+    "molar thermodynamic energy",
+    "angle per time",
+    "dimensionless",
+    "flow coefficient",
+    "standard volume per time",
+    "productivity index",
+    "power",
+    "velocity",
+    "heat transfer coefficient",
+    "density",
+    "thermal conductivity",
+    "specific heat capacity",
+    "emissivity",
+    "volumetric thermal expansion",
+    "dynamic viscosity",
+    "standard volume per standard volume",
+    "mass per mol",
+    "molar volume",
+]
+
+
+def generate_list_of_units(category: str) -> str:
+    """ Return an admonition with toggle to show the units available for a given category. """
+    from barril.units import UnitDatabase
+
+    unit_database = UnitDatabase.GetSingleton()
+    title = unit_database.GetCategoryInfo(category).caption.title()
+    units = unit_database.GetCategoryInfo(category).valid_units_set
```

Comment:
Sort this please (in case `valid_units_set` is a `set`) to ensure a consistent order.

Reply:
line 81 is already sorting the values =)
```
sorted(body, key=str.casefold)

Reply:
Ahh missed that, thanks!